src/iterators/hybrid_reader.c

```diff
@@ -240,13 +228,13 @@ static VecSimQueryReply_Code prepareResults(HybridIterator *hr) {
     return computeDistances(hr);
   }
   // Batches mode.
-  if (hr->child->NumEstimated(hr->child->ctx) == 0) {
+  if (hr->child->NumEstimated(hr->child) == 0) {
```

Comment:
Is this still possible after the optimizations?

Reply:
Not sure if today, but maybe in the future. Also, I wasn't trying to change any logic, only aligning the API. This is not a refactoring of this iterator